index.html

```diff
@@ -297,6 +298,12 @@ <h3>Progress Knight Quest</h3>
                                 <div id="challengeReward4" class="reward">Reward: Increases essence gain by x<span id="challengeEssenceGainBuff">1</span></div>
                                 <button id="challengeButton4" class="w3-button button" onclick="enterChallenge('dance_with_the_devil')">Enter challenge</button>
                             </div>
+                            <div id="legendsNeverDieChallenge" style="padding-top:2em">
+                                <div class="text-caption challenge-title">5. Legends never die</div>
+                                <div style="color: gray; padding-bottom: 0.3em">Reduces your lifespan by ^0.72, happiness does nothing and dark magic XP effect is always 1x</div>

```

Comment:
Lifespan ^0.72 reduces an age-14 start to only having 7 years to live (total lifespan: 21 years).

The minimum exponent on lifespan that can be set in this challenge without immediately dying after respawn is: $$70^x = 14, x = \frac{\log(14)}{\log(70)}$$
which evaluates to approximately 0.62118.

Reply:
Correct! This is kinda hidden from the user but allows them to reach ~175 years